app/src/main/java/com/fstyle/structure_android/screen/main/MainPresenter.java

```diff
@@ -60,21 +61,19 @@ public void searchUsers(int limit, String keyWord) {
         if (!validateDataInput(limit, keyWord)) {
             return;
         }
-        Subscription subscription = mUserRepository.getRemoteDataSource()
-                .searchUsers(limit, keyWord)
-                .subscribeOn(Schedulers.io())
-                .observeOn(AndroidSchedulers.mainThread())
+        Subscription subscription = mUserRepository.getRemoteDataSource().searchUsers(limit,
+                keyWord).subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
                 .subscribe(new Action1<UsersList>() {
-                    @Override
-                    public void call(UsersList usersList) {
-                        mMainView.showListUser(usersList);
-                    }
-                }, new Action1<Throwable>() {
-                    @Override
-                    public void call(Throwable throwable) {
-                        mMainView.showError(throwable);
-                    }
-                });
+            @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 12, expected level should be one of the following: 20, 24.